.github/workflows/test.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
Not even 3.8?

Reply:
If we won't support these versions, I think it's worth a major version bump

Reply:
Also does that mean that our SDK won't support python 3.8? Also might be a reason for a major version bump

Reply:
No tests for Jamba tokenizer?

Reply:
Regarding the version - there was an issue with numpy and the vast support of our Python versions. It was either 3.8 or 3.12

Reply:
Regarding the tests - no. I'm waiting for the vocabs to be OS so I'll be able to test it.